specification/columns/skudetails.md

```diff
@@ -0,0 +1,49 @@
+# SKU Details
+
+The **SKU Details** column represents a list of relevant properties shared by all resources having the same [**SKU ID**](#skuid) which are not captured in another FOCUS column. These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics.
+
+
+The _SkuDetails_ column adheres to the following requirements:
+
+* The properties (both key and value) contained in the _SkuDetails_ column **MUST** be shared across all resources having the same _SkuId_.
+* The _SkuDetails_ column **MUST** be in [_KeyValueFormat_](#key-valueformat).
+* The _SkuDetails_ column **SHOULD NOT** contain properties which are not applicable to the corresponding _SkuId_.
+* Where _SkuDetails_ contains a property with a value which is a numerical, the value provided **MUST** represent the value for a [_ConsumedUnit_](#consumedunit) of 1.
```

Comment:
I don't quite understand this statement.  Is this ensuring that a singular unit is represented as the SKU?

Reply:
Think about an offering that has a unit of measure of 100 bars that has a property that tells me how many foos there 
are. In this case, I don't want the foos per bar; I want the foos per 100 bars since 100 bars is the unit of measure. 

If it helps, think of "bars" as dollars (unit of measure is $100 bills) and "foos" as pennies. When we're talking about a $100 bill, I don't want you to tell me that there are 100 pennies in a dollar; I want you to tell me that there are 10,000 pennies in that $100 bill.

Reply:
Ok maybe a slight rewording will help make this more evident.  Curious how others feel.